Resources/Prototypes/Entities/Markers/Spawners/mobs.yml

```diff
@@ -124,6 +124,32 @@
     prototypes:
       - MobCatFloppa
 
+- type: entity
+  name: Bingus Spawner
+  id: SpawnMobCatBingus
+  parent: MarkerBase
+  components:
+  - type: Sprite
+    layers:
+      - state: green
+      - state: ai
+  - type: ConditionalSpawner
+    prototypes:
+      - MobBingus
+
+- type: entity
+  name: Space Cat Spawner
+  id: SpawnMobCatSpace
+  parent: MarkerBase
+  components:
+  - type: Sprite
+    layers:
+      - state: green
+      - state: ai
+  - type: ConditionalSpawner
+    prototypes:
+      - MobCatSpace
+
```

Comment:
We could either move this to a Nyanotrasen directory, or would you rather try to get this whole PR upstreamed?

Reply:
I PR'd upstream. We'll see what happens.